libs/game/tilemap.ts

```diff
@@ -100,7 +100,12 @@ namespace tiles {
         }
 
         setMap(map: Image) {
-            if (!map) return;
+            if (!map) {
+                this._map = undefined;
```

Comment:
is ``!!this._map`` equivalent to ``this.enabled`` ?

Reply:
ok - is this a scenario?

Reply:
I suppose the ones where I'd want to use it are somewhat better used with ``game.pushScene`` (e.g. a tilemap town where you can enter a house and free roam, or using setTileAt), and otherwise it's not a big cost to just set map to undefined and reset later. I'll remove it